LayoutTests/fast/text/font-face-family.html

```diff
@@ -5,16 +5,16 @@
 </head>
 <body>
 <script>
```

Comment:
Maybe the test should explain the results as a comment on top of the script, so future folks have a bit more context before modifying the test as [293891@main](https://commits.webkit.org/293891@main) did.

Reply:
Could also switch to `js-test.js` while there :)